soh/src/code/code_800EC960.c

```diff
@@ -1671,7 +1679,7 @@ void Audio_OcaSetSongPlayback(s8 songIdxPlusOne, s8 playbackState) {
 
 void Audio_OcaPlayback(void) {
     u32 noteTimerStep;
-    u32 nextNoteTimerStep;
+    u32 nextNoteTimerStep = 0;
```

Comment:
Why is this declaration initialized to zero now?

Reply:
Compiler was warning that it was using an uninitialized variable, and from what I could tell based on the logic, it did look like a real possibility.